docs/perf/baseline-v4.6.1.md

```
@@ -0,0 +1,57 @@
+
+## Latency (curl, local)
+
+| Endpoint | TTFB |
+|---|---:|
+| `GET /` | 19.5 ms |
+| `POST /fire` (p50) | ~2 ms |
+| `POST /fire` (best) | 1.6 ms |
+
+## Font loading strategy
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)